src/megatron/bridge/models/qwen_vl/modelling_qwen3_vl/attention.py

```
@@ -0,0 +1,260 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from megatron.core.transformer.attention import *
```

Ruff (0.15.0)

[error] `from megatron.core.transformer.attention import *` used; unable to detect undefined names

(F403)